Source/Response.swift

```diff
@@ -88,6 +88,12 @@ extension DataResponse: CustomStringConvertible, CustomDebugStringConvertible {
     /// result.
     public var debugDescription: String {
         let requestDescription = request.map { "\($0.httpMethod!) \($0)" } ?? "nil"
+        let parameters = { () -> String in
```

Comment:
You can greatly compact this code, and the naming should reflect the fact that it's the body parameters:
```swift
let body = request?.httpBody.map { String(decoding: $0, as: UTF8.self) } ?? "None"
...
[Request Body]: \(body)
```

Reply:
Thank you for your suggestion